client/layout/index.js

```diff
@@ -24,6 +26,7 @@ import { recordPageView } from 'lib/tracks';
 import TransientNotices from './transient-notices';
 import StoreAlerts from './store-alerts';
 import { REPORTS_FILTER } from 'analytics/report';
+import withSelect from 'wc-api/with-select';
```

Comment:
This `withSelect` can be imported directly from `@wordpress/data` since wc-api is not used. This will work, and its still used in some places when Fresh Data selectors are also used.